website/docs/r/codeartifact_domain.html.markdown

```diff
@@ -30,8 +30,9 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The Name of Domain.
-* `arn` - The ARN of Domain.
+* `id` - The ARN of the Domain.
+* `arn` - The ARN of the Domain.
+* `domain` - The Name of Domain.
```

Comment:
This is defined above as a argument, so doesn't need to be in the attributes section.

```suggestion
```